### PR TITLE
Make new tests

### PR DESCRIPTION
**Pull request by lillian-gardenia-seabreeze**

### CODE REVIEW

**Comment by lillian-gardenia-seabreeze** [comment]
`test_tilemap.py` fails on my laptop, but works on @kcsaff's.

This reason is compounded:

  * Our installation methods may be different, as in I may only be using the `pip install` bitbucket verison, while @kcsaff's may be using the brew installed version, or vice versa. The underlying implications being SDL libraries installed on one machine that differ from the other.
  * pygame.init() is never called, which covers for these discrepancies, wheras, when pygame.init() isn't called, it falls back to more rudimentary SDL usage?

**Comment by lillian-gardenia-seabreeze** [comment]
BREAKING:

I think this will work:

> The other problem you will have once Pygame is working is that images will be corrupted due to a bug/incompatibility with the latest version of SDL_image. You will have to downgrade SDL_image to 1.2.10 as described here:

> https://bitbucket.org/pygame/pygame/issues/284/max-osx-el-capitan-using-the-deprecated

File: tests/test_tilemap.py [hunk 1]
```diff
@@ -39,16 +39,19 @@ def test_subsurface(self):
...
         # then compare it against a master surface to ensure it's the color we
         # want
 
         target_surface = pygame.surface.Surface((1, 1))
         target_surface.blit(tilesheet.tiles[0].image, (0, 0))
+        pygame.image.save(target_surface, 'write-tilesheet-target.png')
 
         master_surface = pygame.surface.Surface((1, 1))
         master_surface.fill((255, 0, 0))
+        pygame.image.save(master_surface, 'write-tilesheet-master.png')
```
Comment (lillian-gardenia-seabreeze) [inline tests/test_tilemap.py]: imma cleanup these then merge! looks good to me 👍 thanks @kcsaff

**Comment by lillian-gardenia-seabreeze** [comment]
The `pygame.image.save` statements should be removed from test tilemap. 

Is it safe to say we can remove `new_tests` and `old_tests`?

**Comment by lillian-gardenia-seabreeze** [comment]
Exploring a possible issue with camera behavior: not centering as expected (see Slack).

**Comment by txanatan** [comment]
retest this please

**Comment by txanatan** [comment]
retest this please

**Pull request merged**